lib/models/game.rb

```diff
@@ -0,0 +1,24 @@
+require 'errors/game_over_error'
+
+module RubyConnect
+  class Game
+    attr_accessor :board, :winning_move
+
+    def initialize(player1, player2)
+      @board = Board.new
+      @winning_move = false
```

Comment:
I'm not sure a predicate method is enough in this class. Would you please show me how you'd do this?